manual/manual/cmds/unified-options.etex

```diff
@@ -70,13 +70,17 @@ The \var{src}".annot" file can be used with the emacs commands given in
 interactively.
 }%notop
 
-\notop{
 \item["-args" \var{filename}]
- Read additional newline-terminated command line arguments from \var{filename}.
-
+Read additional newline-terminated command line arguments from \var{filename}.
+\top{Read additional newline-terminated command line arguments from \var{filename}.
```

Comment:
If I read correctly, the line `Read additional...` will be repeated twice in the toplevel section, as it is present both without any condition and in the `toplevel `conditional. I think that only `It is not possible...` should be there, and similarly for `Read additional... / It is not possible...` below.